scripts/pylint_extensions.py

```diff
@@ -757,6 +757,44 @@ def process_module(self, node):
                         'backslash-continuation', line=line_num + 1)
 
 
+class FunctionArgsOrderChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks the order
```

Comment:
Can you not accommodate more characters in this line? I feel this has been cut-off too soon..